src/Health/HealthController.ts

```diff
@@ -98,6 +107,13 @@ export class HealthController {
     return pickWalletInfoKeys(await this.bitcoinCore.getWalletInfo())
   }
 
+  private async checkWalletBalance(walletInfo: WalletInfo): Promise<WalletInfo> {
+    const { balance } = walletInfo
+    return balance < this.configuration.lowWalletBalanceBTC
```

Comment:
Are we exposing the balance of the wallet? We probably shouldn't. The bool of balanceLow is probably all we should expose.

Reply:
No no, it's ok to expose the balance of the wallet. The node won't be public in any environment for us and shouldn't for anyone else. 

None should set up a node directly against the public internet.

Reply:
Ok sounds good, thanks!